ingestion-beam/src/main/java/com/mozilla/telemetry/amplitude/AmplitudeEvent.java

```diff
@@ -0,0 +1,158 @@
+package com.mozilla.telemetry.amplitude;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.auto.value.AutoValue;
+import com.mozilla.telemetry.util.Json;
+import java.io.Serializable;
+import java.util.Map;
+import javax.annotation.Nullable;
+import org.apache.beam.sdk.coders.Coder;
+import org.apache.beam.sdk.schemas.AutoValueSchema;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.schemas.SchemaCoder;
+import org.apache.beam.sdk.schemas.annotations.DefaultSchema;
+import org.apache.beam.sdk.values.TypeDescriptor;
+
+@AutoValue
+@DefaultSchema(AutoValueSchema.class)
+public abstract class AmplitudeEvent implements Serializable {
+
+  abstract String getUserId();
+
+  abstract String getEventType();
+
+  public abstract long getTime();
+
+  public abstract String getPlatform();
```

Comment:
Is `platform` a required amplitude property?  I was just confused by what this meant at first since it's not namespace

Reply:
Yeah, platform is an Amplitude property. It's not a mandatory one, but namespace is probably the property closest to how AE will interpret platform